test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -10009,6 +10007,60 @@ public void RestoreNetCore_WithMultipleFrameworksWithPlatformAndAssetTargetFallb
             }
         }
 
+        [PlatformFact(Platform.Windows)]
+        public async Task RestoreNetCore_WithCustomAliases_WritesConditionWithCorrectAlias()
+        {
+            // Arrange
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                // Set up solution, project, and packages
+                var solution = new SimpleTestSolutionContext(pathContext.SolutionRoot);
+
+                var packageX = new SimpleTestPackageContext()
+                {
+                    Id = "x",
+                    Version = "1.0.0"
+                };
+
+                var projects = new List<SimpleTestProjectContext>();
+
+                var project = SimpleTestProjectContext.CreateNETCoreWithSDK(
+                    "proj",
+                    pathContext.SolutionRoot,
+                    "net5.0-windows",
+                    "net50");
+
+                // Workaround: Set all the TFM properties ourselves.
+                // We can't rely on the SDK setting them, as only .NET 5 SDK P8 and later applies these correctly.
+                var net50windowsTFM = project.Frameworks.Where(f => f.TargetAlias.Equals("net5.0-windows")).Single();
+                var net50TFM = project.Frameworks.Where(f => f.TargetAlias.Equals("net50")).Single();
+                net50windowsTFM.Properties.Add("TargetFrameworkMoniker", ".NETCoreApp, Version=v5.0");
+                net50windowsTFM.Properties.Add("TargetPlatformMoniker", "Windows, Version=7.0");
+                net50TFM.Properties.Add("TargetFrameworkMoniker", ".NETCoreApp, Version=v5.0");
+                net50TFM.Properties.Add("TargetPlatformMoniker", " ,Version= ");
```

Comment:
" ,Version= " seems really weird, and in my opinion distracts from what the point of this test is supposed to be testing. Doesn't the code handle the case when TPM is not defined, or at worst case `string.Empty`?

Reply:
The current SDK we have for testing sets a default and makes my test cry with a double tfm problem. 
So the problem is that the prop is defined right now.

Reply:
I used android as a test.